docs/data/routes/docs/nextjs/page-routing/switching-to-ssr/en.md

```diff
@@ -3,4 +3,15 @@ name: switching-to-ssr
 routeTemplate: ./data/component-templates/article.yml
 title: Switching to SSR
 ---
-# Walkthrough: Switching to SSR
\ No newline at end of file
+# Walkthrough: Switching to SSR
```

Comment:
Should we spell out the two acronyms or is this super obvious? Maybe a question for @AncaIO  
Disregard this comment if we are covering SSR v SSG in the "why nextjs" page